assets/constants.ts

```diff
@@ -5184,7 +5184,7 @@ export = {
   // This is split up so as to access the viewers in production without
   // exposing the POST and PUT endpoints just yet.
   "ENABLE_NEW_STRUCTURE_PLAYERS": true,
-  "ENABLE_NEW_STRUCTURE_VIEWER_UPDATES": false,
+  "ENABLE_NEW_STRUCTURE_VIEWER_UPDATES": true,
```

Comment:
This is being enabled now, since it  is required for story progress updates.

Reply:
Are there other pages affected by this? Also now that it is enabled, you might want to remove it (and the above flag) altogether

Reply:
The only other area that this was used was in review tests and practice sessions to update skill mastery, which are also required.
Yeah, will remove both these flags soon.